package.json

```diff
@@ -26,6 +24,11 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^3.0.2",
     "eslint-plugin-react": "^6.9.0",
-    "sass-lint": "^1.10.2"
+    "laravel-mix": "^4",
+    "sass": "^1.23.3",
+    "sass-lint": "^1.10.2",
+    "sass-loader": "^7.3.1",
+    "vue-template-compiler": "^2.6.10",
```

Comment:
Is this dependency required by Laravel Mix, given there are no Vue components present?

Reply:
Yes it is. Laravel mix will literally force you to install it if you leave it out.